Modix.Services/Starboard/StarboardHandler.cs

```diff
@@ -84,24 +85,23 @@ private string FormatContent(IUserMessage message, IEmote emote)
         private Embed GetStarEmbed(IUserMessage message)
         {
             var author = message.Author as IGuildUser;
-            var builder = _quoteService.BuildQuoteEmbed(message, author)
+            var embed = _quoteService.BuildQuoteEmbed(message, author)
                 .WithTimestamp(message.Timestamp)
                 .WithColor(new Color(255, 234, 174))
                 .WithAuthor(
                     author.Nickname ?? author.Username,
                     author.GetAvatarUrl() ?? author.GetDefaultAvatarUrl());
 
-            builder.Fields.RemoveAt(builder.Fields.Count-1); //Remove the "Quoted by" field
-            builder.Footer = null;
+            embed.Description = new StringBuilder()
+                .AppendLine($"_Posted in **[#{message.Channel.Name}]({message.GetJumpUrl()})**_")
+                .AppendLine()
+                .AppendLine("**Message**")
+                .AppendLine(embed.Description)
+                .ToString();
 
-            if (message.Embeds.Count == 0)
-            {
-                builder.Description = null;
-                builder.AddField("Message", $"{message.Content}");
-            }
-            builder.AddField("\u200B", $"_Posted in **[#{message.Channel.Name}]({message.GetJumpUrl()})**_");
-            //------------------^ zero-width character
-            return builder.Build();
+            embed.Fields.RemoveAt(embed.Fields.Count-1); //Remove the "Quoted by" field
```

Comment:
Should probably make this cleaner someday, but it should ok for now

Reply:
I mean, yeah, probably, I was contemplating using LINQ to find the field with "Quoted by" in it but I just thought it was excessive when they're always the last thing added (it wouldn't really make sense to put that field anywhere else... Except maybe the top 😬)